_charas/tsuneaki.md

```diff
@@ -48,14 +48,31 @@ h_gachaResult = "Yo, good day.<br>My name is Tsuneaki Toshuku.<br>An exorcist he
 s_gachaResult = "Hm, don't tell me you're…my fan?<br>Heheh, I'm just joking!<br>Uncle's called Tsuneaki.<br>Let's have a long and fun association shall we, Operator-kun?"
 APPRECIATION = "Are you not tired? You don't know when you're tired?<br>Don't slack off with today's work.<br>Afterwards let's go for a date with uncle!...eh, no?"
 DAILY = "Eeh? Do you have some interest with uncle's private life!?<br>Well…if you're fine with not going back, should we spend some time together?"
+EVENTA = "Welcome to Shinraku Festival!<br>I welcome you as the local here.<br>Then let's enjoy this to the fullest, make sure to watch your wallet~!"
+EVENTB = "Arara, Gammei-kun is wearing such an immodest outfit…well no matter.<br>In time of festivals, even outdoor sex is fine, right?"
+EVENTC = "You're so diligent, even taking company paperwork with you on your business trips<br>I'm cheering for you behind the scenes.<br>...eh, uncle has to do it too!?
```

Comment:
Oh, remember that each translation needs to be fully quoted, so this line needs to end with `"` (I will fix in another commit)